developer-docs-site/docs/tutorials/your-first-nft-sdk.md

```diff
@@ -14,18 +14,58 @@ This tutorial describes, in the following step-by-step approach, how to create a
 
 ## Step 1: Pick an SDK
 
+* [Official Aptos Typescript SDK][typescript-sdk]
 * [Official Aptos Python SDK][python-sdk]
-* Official Aptos Typescript SDK -- TBA
 * Official Aptos Rust SDK -- TBA
 
 ## Step 2: Run the Example
 
 Each SDK provides an examples directory. This tutorial covers the `simple-nft` example.
 
-<Tabs>
+Clone `aptos-core`:
```

Comment:
just curious why we add a new doc instead of revising existing first_nft doc using the SDK? eventually, I expect we will only have one first_nft.

Reply:
@davidiw could probably speak more about the decision to write a new tutorial. I can see that the new tutorial is heavily inspired by the old one, but I think the intent is to focus less on theory and internals. I suppose it was easier to write a new tutorial afresh in this case.